exporter/collector/metrics_integration_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integrationtest
```

Comment:
Open to suggestions on a better way to separate this out from the regular unit tests if someone has a better idea

Reply:
See my earlier comment -- perhaps there's no need to separate it, if you can skip (vs failing) integration tests in the absence of credentials.